ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml

```diff
@@ -44,6 +44,10 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+test_binary_build_commands: |
+  mkdir -p /usr/local/bin
+  curl -Lfso /usr/local/bin/jq https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux64
```

Comment:
I think this is shell-like in both this subcommand and the `post.as.commands` below? If so, maybe this could all get wrapped in a short `for` and timer loop (maybe even exponential backoff) so that it's a bit less susceptible to rate-limiting or temporary network problems and maybe the test will be less flaky as a result?